admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/config/TableSettings.scala

```diff
@@ -23,6 +25,7 @@ case class TableSettings(
     hiddenColumns: List[String] = List.empty,
     nonEditableColumns: List[String] = List.empty,
     canBeDeleted: Boolean = true,
+    photoColumn: Option[String] = None,
```

Comment:
I'd do this differently, like `overrideType: Option[CustomDataType]` where `CustomDataType` is an ADT`, it could be a `Text`/`BinaryImage`/`Binary`, this can also handle the `CITEXT` case.

Reply:
Correction, it must be `columnTypeOverrides: Map[String, CustomDataType] = Map.empty`